torch/nn/attention/__init__.py

```diff
@@ -14,11 +14,16 @@
     SDPAParams,
 )
 
+from ._fa4 import flash_attention_fa4_status, register_flash_attention_fa4
+
 
 __all__: list[str] = [
     "SDPBackend",
     "sdpa_kernel",
     "WARN_FOR_UNFUSED_KERNELS",
+    "register_flash_attention_fa4",
```

Comment:
I'm not sure how I feel about these being public here. Since I assume, by design, we will remove them in the future?

What about:
- Make these private
- In a follow up add a "sdpa backend enabler" generic system
- Enable fa4 publically from there

WDYT?
Or is the generic system is a dream and we should just do it this way?